src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -212,16 +234,36 @@ export default class rd2EntryForm extends LightningElement {
 
     /***
     * @description Handle component display when an error occurs
+    * @param error: Error Event
+    * @param disableSaveButton: true to disable the Save button on the page; defaults to false
     */
-    handleError(error) {
-        this.errorMessage = constructErrorMessage(error);
+    handleError(error, disableSaveButton) {
+        this.errorMessage = (!error || (!error.detail && !error.header)) ? constructErrorMessage(error) : error;
```

Comment:
constructErrorMessage(error) returns an Object with Error and Detail instead of the error string. It needs to go one extra step to get the detail from the return object. 
![Screen Shot 2020-06-26 at 2 36 56 PM](https://user-images.githubusercontent.com/57271120/85903281-d004d880-b7ba-11ea-8fcb-0b1ab89a8c40.png)